app/src/main/java/com/ojasvi/memify/MainActivity.kt

```diff
@@ -50,54 +25,17 @@ class MainActivity : ComponentActivity() {
 }
 
 @SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
-@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun MainScreen(modifier: Modifier = Modifier) {
     Scaffold(
-        topBar = { TopAppBar(title = { Text(text = "Memify")})},
-        content = {ImagePicker()},
+        content = { PhotoReasoningRoute() },
```

Comment:
Should be `PhotoReasoningScreen`

Reply:
Pass a modifier to the composable with the inner padding provided by the scaffold

Reply:
Otherwise remove Scaffold, directly call `PhotoReasoningScreen`, and add `Modifier.navigationBarPadding()`

Reply:
Replaced the Scaffold with a box.